galleries/examples/animation/simple_scatter.py

```diff
@@ -31,3 +31,8 @@ def animate(i):
 # ani.save('scatter.gif', writer=writer)
 
 plt.show()
+
+# %%
+#
+# .. tags:: component: animation, plot type: scatter,
+# .. tags:: purpose: reference, level: intermediate
```

Comment:
I think this edit should be made for two reasons:

1) the rendering makes it so that there are two `Tags:` sections in the html

2) the trailing comma makes it treat `""` as a tag which displays oddly and I think is causing the doc build failure.

That said, putting it on one line does break linting due to line length, does the tag directive have a good way of dealing with that/splitting across lines in the same  directive?

Reply:
Thank you for reviewing the commits and the suggestion! I made the change suggested above

Reply:
I think that is a reasonable suggestion. I will open an issue on sphinx-tags and try to get to it. For now can we ignore the linting on this line/file?